native/cocos/application/CocosApplication.cpp

```diff
@@ -52,8 +52,16 @@ int CocosApplication::init() {
     if (_engine->init()) {
         return -1;
     }
-    auto callback = std::bind(&CocosApplication::handleAppEvent, this, std::placeholders::_1); // NOLINT(modernize-avoid-bind)
-    _engine->addEventCallback(OSEventType::APP_OSEVENT, callback);
```

Comment:
Could remove addEventCallback now?

Reply:
It cannot be removed, because it contains not only the state of the engine, but also some other window events that the user can handle. AppEvent can be removed